### ISSUE TITLE
Create Gitter community and chat room

### ISSUE THREAD

**Issue by felipec** [vim/vim#8291]
Conversation was shut down in issue #4738 because it wasn't "on-topic".

Shutting freedom of speech is never a good idea, but it's true a bug report is not the place for discuss other vim issues. Therefore why not create a place where people can speak freely their minds?

Please create a [Gitter](https://gitter.im) chat room under the vim project umbrella.

**Comment by k-takata** [comment]
The vim_dev mailing list has been used for that purpose.

GitHub provides Discussions, but I'm not sure if Bram wants to use it.

**Comment by felipec** [comment]
Yes, but what about the people that want to use something more modern? GitHub allows you to do everything in the browser, presumably some people would want to discuss also in the browser. Mailing lists are too cumbersome for some.

It's much easier to join existing conversations in Gitter.

Just to mention a few projects:

* [docker](https://gitter.im/nodejs/node)
* [travis-ci](https://gitter.im/travis-ci/travis-ci)
* [gitlab](https://gitter.im/gitlab/gitlab)
* [docker](https://gitter.im/docker/docker)

**Comment by Shane-XB-Qian** [comment]
there is a IRC room (compared to gitter) if you want IM chat, you can find it in the vim.org.

**Comment by felipec** [comment]
IRC is not something easily accessible through a browser. There's a reason Gitter was developed.

**Comment by Shane-XB-Qian** [comment]
just saying if you like IM chat, i felt not everyone like IM chat much and/or via a browser, for example: me.
and/but e.g slack or something can work through a browser...
you can create a personal group chat to check how many people like to join... but i knew some people in that vim IRC room.

**Comment by felipec** [comment]
Yes, I can create a personal chat in Discord, but that's not the point.

The point is to have a link in vim.org that people can **click** and join.

Like [this](https://gitter.im/gitlab/gitlab).

**Comment by Shane-XB-Qian** [comment]
i think there is No such rule in the world saying: if github or "new" style had something, then vim org had to have something too...
specially i felt gitter would not improve something too much if compared to IRC or current style..
besides/and there is no biz company behind vim org, here just some "boring" person, doing some "boring" things, offering warm heart to world.. :-)

**Comment by chrisbra** [comment]
I am not against the idea in general, just pointing out some alternatives: [reddit](https://www.reddit.com/r/vim/), [stackexchange](https://vi.stackexchange.com/), vims irc channeld used to be available on freenode with a web interface (I suppose after all the freenode fuss, that libera will offer something similar) and the vim mailinglists can also be used from within a browser using the google groups webinterface.

**Comment by mcepl** [comment]
1. Gitter has been acquired by Element, I would go straight for Matrix, not bothering with Gitter on the way.
2. Every matrix instance (including the main https://app.element.io) has a bridge to IRC, so `#vim` would be available, except it is now lost in [the craziness of Freenode falling apart](https://gist.github.com/joepie91/df80d8d36cd9d1bde46ba018af497409).
3. There is `#vim:matrix.org`, which doesn’t seem to be too active, and it certainly doesn’t have any label of officiality.

So, in conclusion, I would suggest to declare `#vim:matrix.org` as the official Vim chatroom.

**Comment by phy1729** [comment]
The freenode channel has moved to Libera.Chat. There's a web interface at https://web.libera.chat/#vim .

**Comment by brammool** [comment]
Feel free to pick some place to discuss user issues.  But for development discussions email is the only way I can keep things organised.  And github issues work for bugs and changes (since it's copied to the maillist).
I already struggle to keep up with things using email, I don't really need another communication channel to use.

**Comment by felipec** [comment]
> The freenode channel has moved to Libera.Chat. There's a web interface at https://web.libera.chat/#vim .

That is very nice indeed.

Unfortunately you can't see the log of previous discussions like in Gitter, but that's good enough for me, and I suspect most people.

Could we add a link to that in the vim.org website?

**Comment by felipec** [comment]
> Feel free to pick some place to discuss user issues.

Yes, the suggestion was for us mere mortals to discuss things that are potentially off-topic in other venues.

Any serious proposal would have to go through official channels.

**Comment by mcepl** [comment]
@felipec And of course, you can always come to [the Dark Side](https://neovim.io/), we have cookies, [working git](https://github.com/neovim/neovim), [chat](https://gitter.im/neovim/neovim) and [discourse](http://discourse.neovim.io/) (not sure why).

**Comment by felipec** [comment]
Do you care to explain why was this closed?

**Comment by chrisbra** [comment]
sure, it looks like this is not needed. It has been pointed out that there are many alternatives already where you can discuss about Vim.

**Comment by felipec** [comment]
Yes, and the disadvantages of **all** those alternatives have been pointed out as well.

There isn't a single alternative that provides all what Gitter provides.

**Comment by chrisbra** [comment]
> Yes, and the disadvantages of all those alternatives have been pointed out as well.

I assume, you mean that you need an account for those services? That is true, but you need one for gitter as well, even if can authorize against github (which means you still need a github account). I don't see that as a clear advantage. There are plenty of discussion forums, just use any of those or setup a discourse server. I don't see this as an issue of vim.

**Comment by felipec** [comment]
@chrisbra What prompted this request is that some comments where removed from #4738.

We need a different place where people can discuss freely, instead of people commenting on GitHub issues.

The relevant people **already** have GitHub accounts.

**Comment by chrisbra** [comment]
I was going to ignore this thread, but I need to clarify one final thing:

> @chrisbra What prompted this request is that some comments where removed from #4738.

Nothing was removed, just marked as off-topic (which it was).

**Comment by felipec** [comment]
> I was going to ignore this thread

Saying "I'm going to mute you" is **bad manners**, just mute and move.

But at this point it's very clear the members of the vim project are not interested in doing things right.

**Comment by brammool** [comment]
We are very much interested in doing things right.  But what we call right is apparently differing from your opinion.

**Comment by felipec** [comment]
> We are very much interested in doing things right.

Do you think it's right to:

1. Mark some comments as off-topic
2. **Close** a request to open another channel of communication for people that want to discuss off-topic things
3. Provide a substandard reasoning for closing the issue
4. **Ignore** requests for a good explanation

**Comment by brammool** [comment]
Issues in github are for bugs and feature request of the Vim code.  Not for a discussion about communication, it won't change anything in what is distributed.

**Comment by felipec** [comment]
> Issues in github are for bugs and feature request of the Vim code.

Really? Then why do you have labels for "question", "discuss", and "website"?

Moreover, @chrisbra didn't immediately close the issue because it "didn't belong in GitHub", he used a different rationale which is not valid.

And you didn't answer the question: do you think it's OK for him to close an issue with a rationale that isn't true, and then state he will **ignore** further requests for explanation?

**Comment by k-takata** [comment]
> Really? Then why do you have labels for "question", "discuss", and "website"?

Even GitHub issues are used for bugs and features, still some people write questions, or sometimes the issue turns out that it is caused by a mistake (or misunderstanding) of the user.
In such cases, I label them "question" so that someone can easily ignore them when they search about a bug.
